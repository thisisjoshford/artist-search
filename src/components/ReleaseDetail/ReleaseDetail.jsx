import React from 'react';
import PropTypes from 'prop-types';
// import { Link } from 'react-router-dom';


const ReleaseDetail = ({ artistName, ReleaseDetail }) => {
  // const artistReleases = ReleaseDetail.map((release) => (
  //   <h3>hi</h3>
  // ));

  return (
    <>
      <h3>Releases for: {artistName}</h3>
      {/* {artistReleases} */}
    </>
    
  );
};

ReleaseDetail.propTypes = {
  artistName: PropTypes.string.isRequired,
  ReleaseDetail: PropTypes.array.isRequired
};

export default ReleaseDetail;
