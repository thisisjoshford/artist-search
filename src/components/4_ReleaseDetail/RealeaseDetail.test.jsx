import React from 'react';
import { shallow } from 'enzyme';
import ReleaseDetail from './ReleaseDetail';

describe('Search component', () => {
  it('renders Search', () => {
    const releaseDetail = [
      {
        id: '1234',
        title: 'track1',
      },
      {
        id: '12345',
        title: 'track2'
      }
    ];

    const wrapper = shallow(
      <ReleaseDetail artistName='Tool' albumName='Album1' releaseDetail={releaseDetail}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
