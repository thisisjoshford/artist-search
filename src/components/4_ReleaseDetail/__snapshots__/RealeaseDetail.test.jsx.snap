// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders Search 1`] = `
<Fragment>
  <h3>
    Album1
     Track List:
  </h3>
  <Link
    key="1234"
    to="/lyrics/Tool/track1"
  >
    <h3>
      track1
    </h3>
  </Link>
  <Link
    key="12345"
    to="/lyrics/Tool/track2"
  >
    <h3>
      track2
    </h3>
  </Link>
</Fragment>
`;
